packages/@lwc/template-compiler/src/codegen/index.ts

```diff
@@ -77,6 +77,7 @@ import {
 import { format as formatModule } from './formatters/module';
 
 function transform(codeGen: CodeGen): t.Expression {
+    const instrumentation = codeGen.state.config.instrumentation;
```

Comment:
Grab the instrumentation object from the state here so we can call it from any of the subsequent functions as `instrumentation` without having to dereference it multiple times.